cluster/validation.go

```diff
@@ -731,3 +739,19 @@ func getClusterVersion(version string) (semver.Version, error) {
 	}
 	return parsedVersion, nil
 }
+
+// warnWeaveDeprecation prints a deprecation warning if version is higher than 1.27
+func warnWeaveDeprecation(k8sVersion string) error {
+	version, err := util.StrToSemVer(k8sVersion)
+	if err != nil {
+		return fmt.Errorf("failed to translate: %s to semver notation: %v", k8sVersion, err)
+	}
+	version127, err := util.StrToSemVer("v1.27.0")
+	if err != nil {
+		return fmt.Errorf("fialed to tranlate v1.27.0 to semver notation: %v", err)
```

Comment:
Minor typo and error wrapping (if we can do it):
```suggestion
		return fmt.Errorf("failed to tranlate v1.27.0 to semver notation: %w", err)
```